app/models/work.rb

```diff
@@ -0,0 +1,8 @@
+class Work < ApplicationRecord
+  has_many :votes
+  CATEGORIES = %w(album book movie)
```

Comment:
Nice job with this, you can actually leverage this EVEN MORE! Think about what this gives you on your works/index page!